packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -18,10 +18,14 @@ type CreateTeamsOptions = {
 export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) => {
   const { teamNames, orgId } = input;
 
-  const organization = await prisma.team.findFirst({ where: { id: orgId }, select: { metadata: true } });
+  const organization = await prisma.team.findFirst({
+    where: { id: orgId },
+    select: { slug: true, metadata: true },
+  });
   const metadata = teamMetadataSchema.parse(organization?.metadata);
 
-  if (!metadata?.requestedSlug) throw new TRPCError({ code: "BAD_REQUEST", message: "no_organization" });
+  if (!metadata?.requestedSlug || !organization?.slug)
+    throw new TRPCError({ code: "BAD_REQUEST", message: "no_organization_slug" });
 
   const userMembership = await prisma.membership.findFirst({
     where: {
```

Comment:
Not directly related to this PR, but this needs a check to confirm the membership isn't pending